components/intro-template.tsx

```diff
@@ -0,0 +1,139 @@
+import Link from 'next/link'
+
+export default function IntroTemplate() {
+  return (
+    <div className="my-16 md:mb-28 md:grid md:grid-cols-2 md:gap-x-16 lg:gap-x-16">
+      <div>
+        <iframe
```

Comment:
To double check, we don't want any sort of analytics apart from the ones we get from youtube for this round right?

Reply:
I talked to Jon and no - no tracking necessary for this 😄

Reply:
Niceee ✨